kumascript/package.json

```diff
@@ -21,7 +21,7 @@
     "jest-junit-reporter": "^1.1.0",
     "jsdom": "^11.12.0",
     "pegjs": "^0.10.0",
-    "vnu-jar": "^18.12.18"
+    "vnu-jar": "^20.3.16"
```

Comment:
@escattone Are we even running the unit tests for the `kumascript` workspace?
Before you answer, and I think you agree with me, I'd rather we focus on a more "holistic" approach of "Does it build?™" with our `scripts/ci-end-to-end.sh` and perhaps our headless tests (which, I guess, will also be part of the `script/ci-end-to-end.sh`) 

For the record, I see it being used here: https://github.com/mdn/stumptown-renderer/blob/9ae77269dc8eb99110f6b5c0d4168702603c3200/kumascript/tests/macros/utils.js#L9

Reply:
@peterbe I've kept the `kumascript` unit tests up-to-date, so they run successfully, but I haven't yet added them to the CI tests, which I think we should do, since they are valuable at least until we've developed really good end-to-end tests.

Reply:
Yes, please! Actually, it would make perfect sense to have both. In any platform. 

I would argue that, to some degree, we're hoping the using TypeScript will prevent some bugs from going into prod on the `client` but it's not the only thing we're doing, right? 

Actually, that reminds me, until we have...

* Documentation/script on how to continually port/copy the macros from mdn/kumascript into this repo.
* Run the kumascript "unit tests" in the Yari CI

...can we call the KS-rendering done. (A little depressing maybe?)

Reply:
@peterbe 👍 No we can't! 😄 Now that https://github.com/mdn/stumptown-renderer/issues/567 is closed, I just need to add some kumascript documentation and also create a script for updating/copying macros. I'll create an issue for that so I don't forget.

Reply:
Created https://github.com/mdn/stumptown-renderer/issues/569 and https://github.com/mdn/stumptown-renderer/issues/570